l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -425,6 +425,7 @@
                         <field name="fatturapa_payments" nolabel="1" readonly="1">
                             <tree string="Payments">
                                 <field name="payment_terms" />
+                                <field name="payment_methods" widget="many2many_tags" />
```

Comment:
> Il widget da solo non è molto utile perchè mostra dati tecnici:
> ![Screenshot from 2021-09-23 15-12-36](https://user-images.githubusercontent.com/1033131/134513754-4771410f-4f14-4020-89ac-49e83a8ecc1e.png)
> 
> Ho portato la modifica su 12 #2433 aggiungendo `_rec_name`:
> ![Screenshot from 2021-09-23 15-13-25](https://user-images.githubusercontent.com/1033131/134513845-768fa5de-7bfb-477f-b15f-44146c06e343.png)

(da https://github.com/OCA/l10n-italy/pull/2369#discussion_r714786756)
Ora abbiamo due possibilità:

- rimuovi il widget da questa PR e la modifica sarà in v14 quando qualche buonanima porterà #2433 a v14
- porti la modifica di #2433 (al momento è ancora aperta) in questa PR

Purtroppo in entrambi i casi c'è da modificare qualcosa qui :)